### PR TITLE
[ENH] Add Travis CI

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
No Travis and therefore no testing.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by kernc** [comment]
Seems to fail. Widgets/tests need Orange. Install that in _.travis.yml_ as well.

**Comment by jerneju** [comment]
@kernc : Now it works!

File: setup.py [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 VERSION = '1.1.5'
 
+INSTALL_REQUIRES = (
+    'Orange3>=3.9.0',
```
Comment (kernc) [inline setup.py]: No, sorry, install Orange on Travis only. I promote this package as a generally useful association rules mining package. No need to tag the whole Orange and company everywhere.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+before_install:
+    - pip install -U setuptools pip wheel
+    - pip install codecov
+
+install:
+    - pip install PyQt5 AnyQt
+    - travis_wait pip install -e .
+
+script:
+    - catchsegv xvfb-run -a -s "-screen 0 1280x1024x24" coverage run setup.py test
```
Comment (kernc) [inline .travis.yml]: AFAIU, we have zero tests and this actually does nothing?

Comment (jerneju) [reply .travis.yml]: Tests will be added in a separate PR.

**Pull request merged**